### ISSUE TITLE
Add `shuffle: bool` parameter in `train_test_split`

### ISSUE THREAD

**Issue by deependujha** [Lightning-AI/litData#673]
## 🚀 Feature

`train_test_split` function randomly shuffles the dataset and then splits it.

- Add a parameter `shuffle: bool`, which can be used to split without shuffling.

### Motivation

check this [thread](https://discord.com/channels/1077906959069626439/1402295130391576747)

**Comment by otogamer** [comment]
I have created a PR. Please review it. Thank you in advance.
